packages/grafana-ui/src/components/TimePicker/TimeOfDayPicker.tsx

```diff
@@ -10,16 +12,65 @@ interface Props {
 }
 
 export const TimeOfDayPicker: FC<Props> = ({ minuteStep = 1, showHour = true, onChange, value }) => {
+  const theme = useContext(ThemeContext);
+
+  const popupStyles = css`
+    .rc-time-picker-panel-select {
```

Comment:
Hi @peterholmberg, thank you for the review! Do you know why we're not using React-specific `@emotion/core`, which should handle all this theming (at least from my experience with [styled-components](https://styled-components.com/), could be different with emotion) without extra wrappers?

Reply:
Another question: 
I'm trying to use `stylesFactory` but getting an error ```TypeError: Object(...) is not a function
    at Module../src/components/TimePicker/TimeOfDayPicker.tsx (http://localhost:9001/main.0027a4de5f159f14c56f.bundle.js:39403:77)```.

Do you know what I could be missing here?
```typescript
const getStyles = stylesFactory((theme: GrafanaTheme) => {
  return {
    caretWrapper: css`
      position: absolute;
      right: 8px;
      top: 50%;
      transform: translateY(-50%);
      display: inline-block;
      text-align: right;
      z-index: 1071;
    `,
    picker: css`
      .rc-time-picker-panel-select {
        border-color: ${theme.colors.dark6};
        font-size: 14px;
        li {
          outline-width: 2px;
          &.rc-time-picker-panel-select-option-selected {
            background-color: inherit;
            border: 1px solid ${theme.colors.orange};
            border-radius: ${theme.border.radius.sm};
          }
        }
      }
    `,
  };
});

export const TimeOfDayPicker: FC<Props> = ({ minuteStep = 1, showHour = true, onChange, value }) => {
  const theme = useTheme();
  const styles = getStyles(theme);

  return (
    <div>
      <RcTimePicker
        popupClassName={styles.picker}
        defaultValue={dateTimeAsMoment()}
        onChange={(value: any) => onChange(dateTime(value))}
        allowEmpty={false}
        showSecond={false}
        value={dateTimeAsMoment(value)}
        showHour={showHour}
        minuteStep={minuteStep}
        inputIcon={<Caret wrapperStyle={styles.caretWrapper} />}
      />
    </div>
  );
};
```

Reply:
Solved the issue by importing `stylesFactory` from `'../../themes';` instead of `import { useTheme, Icon } from '../../index';`, which doesn't work for some reason.